models/zeroex/polygon/zeroex_api_polygon_fills.sql

```diff
@@ -387,4 +387,4 @@ AND mp.minute >= '{{zeroex_v3_start_date}}'
 {% endif %}
 
 LEFT OUTER JOIN {{ ref('tokens_erc20') }} ts ON ts.contract_address = taker_token and ts.blockchain = 'polygon'
-LEFT OUTER JOIN {{ ref('tokens_erc20') }} ms ON ms.contract_address = maker_token and ms.blockchain - 'optimism'
\ No newline at end of file
+LEFT OUTER JOIN {{ ref('tokens_erc20') }} ms ON ms.contract_address = maker_token and ms.blockchain = 'optimism'
```

Comment:
@RantumBits I was thinking `ts` and `ms` tables are across chains so I did not change the 'optimism' to 'polygon'.